src/app.ts

```diff
@@ -0,0 +1,16 @@
+import express from 'express'
+import compression from 'compression'
+import cookieParser from 'cookie-parser'
+import dotenv from 'dotenv'
+import router from './routes'
+
+dotenv.config()
+const app = express()
+app.disabled('x-powered-by')
+app.set('port', process.env.PORT ?? 3000)
+app.use(express.json())
+app.use(express.urlencoded({ extended: false }))
+app.use(compression())
+app.use(cookieParser())
+app.use(router)
```

Comment:
```suggestion
app.use('/api/v1', router);

if (NODE_ENV === 'production') {
  app.use(express.static(join(__dirname, '..', 'client', 'build')));

  app.get('/*', (req, res) => {
    res.sendFile(join(__dirname, '..', 'client', 'build', 'index.html'));
  });
}
```

Reply:
and try to organize the file a little bit more by adding empty line between the middleware and the router for example, and move the `app.set` to the end just before the router